src/components/sections/homepage/hero/index.tsx

```diff
@@ -120,7 +119,15 @@ export const Hero = () => {
           />
 
           <div className={s['ctas']}>
-            <VideoModal />
+            <VideoModal
+              poster="/images/homepage/hero-video-placeholder.png"
+              url="https://stream.mux.com/j4HHD01eAGd01vFBPyQhfpWuDWSlXKsGep1o2dbDAlE2s.m3u8"
+              button={
```

Comment:
I would do this 👇 like to make the component more flexible and maybe having multiple triggers

```
<VideoModal
 poster="/images/homepage/hero-video-placeholder.png"
 url="https://stream.mux.com/j4HHD01eAGd01vFBPyQhfpWuDWSlXKsGep1o2dbDAlE2s.m3u8"
>
  <VideoModal.Trigger>
    <Button>Bla bla</Button>
  </VideoModal.Trigger>
<\VideoModal>
```